### PR TITLE
Implementing organization column in newsflash.

### PR DESCRIPTION
**Pull request by omerinvia**

### CODE REVIEW

**Comment by atalyaalon** [comment]
@omerinvia great work!
After we'll insert this commit - we'll want to backfill to the existing news flashes, and then replace the source_mapping in flask_app.py (see [here](https://github.com/hasadna/anyway/pull/1365))

@Mano3 - with this pr - is it possible to backfill the organization column for all news flashes?

**Comment by atalyaalon** [comment]
> @omerinvia great work!
> After we'll insert this commit - we'll want to backfill to the existing news flashes, and then replace the source_mapping in flask_app.py (see [here](https://github.com/hasadna/anyway/pull/1365))
> 
> @Mano3 - with this pr - is it possible to backfill the organization column for all news flashes?

BTW, even if the answer is no, I believe we can insert this pr once @Mano3 approves, and then add another pr for the above.

**Comment by omerinvia** [comment]
Thanks for the fast commenting :) While I tried to backtrack how things are working it looks like theres an api which update the newsflashes so I put a line of code which should migrate the organizations as well.

File: anyway/flask_app.py [hunk 1]
```diff
@@ -75,12 +75,14 @@
 from anyway.infographics_utils import get_infographics_data
```
[the remark this answers is not in the document]
Comment (omerinvia) [reply anyway/flask_app.py:75]: its part of the BLACK formatting :)

File: anyway/infographics_utils.py [hunk 2]
```diff
@@ -491,11 +491,11 @@ def get_latest_accident_date(table_obj, filters):
```
[the remark this answers is not in the document]
Comment (omerinvia) [reply anyway/infographics_utils.py:492]: its part of the BLACK formatting :)

File: anyway/parsers/news_flash.py [hunk 3]
```diff
@@ -52,6 +58,7 @@ def scrape_extract_store_twitter(screen_name, db):
             # We can break if we're guaranteed the order is descending
             continue
         newsflash.accident = classify_tweets(newsflash.description)
+        newsflash.organization = classify_organization("twitter")
```
[the remark this answers is not in the document]
Comment (omerinvia) [reply anyway/parsers/news_flash.py:61]: in the feature description there was a required mapping which implemented I think that the idea that when we will have more sophisticated mapping (which is not straightforward) we will implement this `classify_organi..` call

[the remark this answers is not in the document]
Comment (atalyaalon) [reply anyway/parsers/news_flash.py:61]: @omerinvia Great work and welcome to our project! You are welcome to [join our slack](https://join.slack.com/t/hasadna/shared_invite/zt-gzb80b42-C2Kw4xE2tLThF_Jx303Cgw)

@Mano3 For example, we want import from the same source but various organizations (twitter: both מד״א and איחוד הצלה we would like to save in the db both the source (twitter) and the organization of the tweet (either מד״א or איחוד הצלה).
So right now it's a "stupid" mapping, but soon it won't be.
Also - we might have the same organization but different sources (מד״א - twitter and perhaps another source, for example data from their system).

What do you think?

[the remark this answers is not in the document]
Comment (Mano3) [reply anyway/parsers/news_flash.py:61]: Only a many to many scenario justifies this in my opinion. So I approve.

**Review by Mano3 — APPROVED** [review]

**Pull request merged**